drivers/hmis/app/models/hmis/workflow_definition/flow.rb

```diff
@@ -7,5 +7,18 @@ class Flow < GrdaWarehouseBase
     belongs_to :template, class_name: 'Hmis::WorkflowDefinition::Template'
     belongs_to :source_node, class_name: 'Hmis::WorkflowDefinition::Node'
     belongs_to :target_node, class_name: 'Hmis::WorkflowDefinition::Node'
+
+    def describe_as_string(source_only: false, target_only: false)
```

Comment:
This is fine for a dev-facing function. A tiny nit is to try to avoid mutually exclusive flags as it creates ambiguity; it's better if invalid states cannot be represented; for example by rolling them into a single state param. But this is just an observation, no change is needed